crm_phonecall_timesheet/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2015 Javier Iniesta <javieria@antiun.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "CRM Timesheet",
+    'category': 'Customer Relationship Management',
+    'version': '10.0.1.0.0',
+    'depends': [
+        'crm',
+        'hr_timesheet',
+        'crm_timesheet',
```

Comment:
It was removed on my commit: https://github.com/OCA/hr-timesheet/pull/106/commits/3da7180da6a150e97b9d598e80bc175c7dfdd891
But GitHub didn't outdate this...